File: Logica de Programação/Equacao.java
public class Equacao {
    public static void main(String[] args) {
        // Inicializa a variável i com 2
        int i = 2;
        
        // Calcula a raiz quadrada de i usando Math.sqrt()
        double r = Math.sqrt(i);
        
        // Exibe o resultado em partes, usando print e println
        System.out.print("A raiz quadrada de ");
        System.out.print(i);
        System.out.print(" é ");
        System.out.print(r);
        System.out.println(".");

        // Atualiza i para 5 e recalcula a raiz quadrada
        i = 5;
        r = Math.sqrt(i);

        // Exibe o resultado usando concatenação direta na string
        System.out.println("A raiz quadrada de " + i + " é " + r + ".");
    }
}

File: Logica de Programação/Equacao.py
import math  # Importa o módulo math para funções matemáticas

# Solicita um número inteiro ao usuário
n1 = int(input('Insira um valor: '))

# Calcula a raiz quadrada usando potência (exponenciação)
raiz_quadrada = n1 ** (1/2)
print(raiz_quadrada)

# Calcula a raiz quadrada usando a função sqrt() do módulo math
metodo_2 = math.sqrt(n1)
print(metodo_2)
